pkg/platform/yaml/testdb/util.go

```diff
@@ -246,9 +246,9 @@ func Decode(yamlTestcase *yaml.NetworkTrafficDoc, logger *zap.Logger) (*models.T
 		switch reflect.ValueOf(httpSpec.Assertions["noise"]).Kind() {
 		case reflect.Map:
 			for k, v := range httpSpec.Assertions["noise"].(map[string]interface{}) {
-				tc.Noise[k] = []string{}
+				tc.Noise[strings.ToLower(k)] = []string{}
 				for _, val := range v.([]interface{}) {
-					tc.Noise[k] = append(tc.Noise[k], val.(string))
+					tc.Noise[strings.ToLower(k)] = append(tc.Noise[k], val.(string))
```

Comment:
shouldn't the tc.Noise[k] inside append be lower case too..? may be create a variable which represents k as lower case and use it this case.